source/extensions/tracers/zipkin/zipkin_core_types.h

```diff
@@ -6,13 +6,15 @@
 #include "envoy/common/time.h"
 #include "envoy/network/address.h"
 
+#include "common/common/empty_string.h"
```

Comment:
we should not put any empty strings in protobuf messages, unless it is a tag and the value was intentionally set to empty

Reply:
What would be the use case for an empty tag? `error` maybe?

Reply:
usually that's the only use case yes

Reply:
Uhmm, have mixed feelings here. Why would you tag a span with error but not saying what the error is?